NEWS.md

```diff
@@ -1,3 +1,7 @@
+# sircovid 0.12.0
+
+* Introduced `p_NC`, `p_NC_weekend`, `phi_pillar2_cases` and `phi_pillar2_cases_weekend` parameters by age groups. These are used for age-specific fits to pillar 2 positivity and cases, respectively
```

Comment:
I think it would be better to frame the news more about being able to fit to the pillar 2 age groupings, with the introduction of the extra parameters as secondary

Reply:
updated